src/api/test-mocks/mock-application.ts

```diff
@@ -142,13 +142,25 @@ export const mockLossPayeeFinancialDetailsUk = {
   bankAddress: 'Mock UK financial address',
 };
 
+export const mockLossPayeeFinancialDetailsInternational = {
+  iban: 'AAaaa1A1AAaaAAa1AaAaAaaaAaAaaaAaAAAaAaAaAaA=',
+  ibanVector: 'AAaaAAA1aA+1aaa1',
+  bicSwiftCode: 'BBbbb2B1BBbbBBb2BbBbBbbbBbBbbbBbBBBbBbBbBbB',
+  bicSwiftCodeVector: 'BBbbBBB2bB+2bbb2',
```

Comment:
Could we have different values to the UK details?

Reply:
in next PR